pkg/object/httppipeline/httppipeline.go

```diff
@@ -69,7 +68,7 @@ type (
 	// Spec describes the HTTPPipeline.
 	Spec struct {
 		Flow    []Flow                   `yaml:"flow" jsonschema:"omitempty"`
-		Filters []map[string]interface{} `yaml:"filters" jsonschema:"-"`
+		Filters []map[string]interface{} `yaml:"filters" jsonschema:"required"`
```

Comment:
`flow` should be `required` as `filters` is `required`

Reply:
Nope, `flow` is allowed to be empty, which cause the all `filters` will be a normal pipeline without jumping.